recipes/kartothek/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "kartothek" %}
+{% set version = "3.0.0" %}
```

Comment:
Do we need to bump this for every release or is this automated jinja magic based on the repo tags?

Reply:
There's a bot that will make a PR on each release. We just need to click `Merge`.